src/pages/page-names.constants.ts

```diff
@@ -42,4 +42,5 @@ export const CAT_BE = {
   REKEY_UPLOAD_OUTCOME_PAGE: 'RekeyUploadOutcomePage',
   NON_PASS_FINALISATION_PAGE: 'NonPassFinalisationCatBEPage',
   VEHICLE_CHECKS_MODAL: 'VehicleChecksCatBEModal',
+  VIEW_TEST_RESULT_PAGE:'ViewTestResultCatBPage',
```

Comment:
Cat B page not BE

Reply:
Cat BE page doesn't exist yet so will update this again when I add it .